documentation/shared/attributes.adoc

```diff
@@ -42,14 +42,22 @@
 :kafka-exporter-project: link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter^]
 
 //OAuth attributes and links
+:strimzi-project-name: Strimzi
+:keycloak-project-name: Keycloak
 :oauth2-site: link:https://oauth.net/2/[OAuth 2.0 site^]
 :oauth-version: 0.7.1
 :keycloak-server-doc: link:https://www.keycloak.org/documentation.html[Keycloak documentation^]
+:keycloak-server-install-doc: link:https://www.keycloak.org/docs/latest/server_installation/index.html#_operator[Installing the Keycloak Operator^]
 :keycloak-authorization-services: link:https://www.keycloak.org/docs/latest/authorization_services/index.html[Authorization Services^]
+:authz-services-model-ref: xref:con-oauth-authorization-services-model-{context}[Keycloak Authorization Services model for managing permissions]
 :oauth-blog: link:https://strimzi.io/2019/10/25/kafka-authentication-using-oauth-2.0.html[Kafka authentication using OAuth 2.0^]
-:oauth-demo-keycloak: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/{oauth-version}/examples[Using Keycloak as the OAuth 2.0 authorization server^]
-:oauth-demo-hydra: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/{oauth-version}/examples/docker[Using Hydra as the OAuth 2.0 authorization server^]
-:oauth-kubernetes-examples-link: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/{oauth-version}/examples/kubernetes[https://github.com/strimzi/strimzi-kafka-oauth/tree/{oauth-version}/examples/kubernetes]
+:oauth-demo-keycloak: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/master/examples[Using Keycloak as the OAuth 2.0 authorization server^]
+:oauth-demo-hydra: link:https://github.com/strimzi/strimzi-kafka-oauth/tree/master/examples/docker[Using Hydra as the OAuth 2.0 authorization server^]
```

Comment:
Should these really point to master?
* `master` does not exist anymore
* It should probably point to some release tag

Reply:
Good catch about master. But otherwise yes, I think it's best to refer to 'main' here, for the latest docs.

Reply:
Why is it best to refer to unreleased / unused documentation / examples?

Reply:
In my experience the latest documentation is the most polish, with bugs fixed, better explanations added ...

Reply:
That might be valid now. But next month it might be that the main documentation contains some new stuff which does not apply. If what you say was the only thing which mattered, we would have only master documentation and nothing else and let the users figure out what is valid and what is not. But that is not how it works normally and I do not think we should start with that.

Reply:
Ok, so we need a release-time replacing of links where:
- for documentation links to strimzi-kafka-oauth project we have to replace `/main/` in the link with release tag (e.g. `/0.7.1/`)
- for examples README.md links to strimzi.io we have to replace `/latest/` in the link with the release tag (e.g. `0.23.0`)

So the question is what is the process of releasing - what script with what command is executed as a pre-release that prepares docuemntation and examples release artifacts before they are committed, tagged, and pushed. The replacing logic should be part of that make target / script.

Reply:
I found [development-docs/RELEASE.md](https://github.com/strimzi/strimzi-kafka-operator/blob/main/development-docs/RELEASE.md).

Reply:
In documention, you should use the variable which is already there. So I think it is just the README file you are adding.

Reply:
In the release makefile targets there are already different replacements ... so you just need to add one more for this file I guess.